backend/src/server/routes/v1/organization-router.ts

```diff
@@ -108,6 +109,7 @@ export const registerOrgRouter = async (server: FastifyZodProvider) => {
     config: {
       rateLimit: readLimit
     },
+    preHandler: extendTimeout(QUERY_TIMEOUT),
```

Comment:
I think we should find out the query that causing the latency rather than increasing the timeout.